README.md

```diff
@@ -0,0 +1,58 @@
+# CometBFT Knowledge Base
+
+[![Creative Commons Attribution 4.0 International
+License][license-img]][license]
+
+This repository is intended to act as a team [Zettelkasten][zk] for the CometBFT
+core team as an attempt to preserve both knowledge gained and insights gleaned
+by the team - both for the current team, and future people working on the
+project.
+
+It is _always_ a work-in-progress. It is made public in case other people or
+users may find it to be of use/interest.
+
+## Requirements
+
+At present, [Foam] is used for editing and navigating the content - this may
+change in future as the team explores different tools for managing the knowledge
+base.
+
+## Structure
+
+This knowledge base is very loosely organized, and is intended to be seen as a
+graph of interrelated notes/ideas/concepts.
+
+- The directory structure does not matter very much, as all content is loaded
```

Comment:
My thinking here is on the opposite direction: if the content is produced/navigated using the tool, then the directory structure does not matter much.